README.md

```diff
@@ -44,6 +44,18 @@ $ npm install # optional: install CLI dependencies
 $ sudo npm link # optional: link CLI for system-wide execution
 ```
 
+## Update
+
+```bash
+$ mn stop
+$ git checkout master
+$ git pull
+$ mn update
+$ mn start
+```
+
+If the platform layer has been wiped, you must additionally run `$ mn reset --platform-only --hard` followed by `mn setup -k <bls-key>` with your existing BLS private key after the first `mn stop` command.
```

Comment:
Will the setup work if we reset the platform only?

Reply:
Actually I'm not sure if `mn setup` is necessary again. Is it possible for the node to use the same `validatorKey` and `nodeKey` on the new network? Would something like this work instead?

```bash
$ mn stop
$ git checkout master
$ git pull
$ mn reset --platform-only
$ mn update
$ mn start
```

I think the other config values (`genesis`, `validators`, `contracts`) would be overwritten by the new values from the `git pull` command? Or would we need to write migrations for this?